targets/TARGET_STM/rtc_api_hal.h

```diff
@@ -66,11 +66,7 @@ extern "C" {
 #if DEVICE_LPTICKER && !MBED_CONF_TARGET_LPTICKER_LPTIM
 /* PREDIV_A : 7-bit asynchronous prescaler */
 /* PREDIV_A is set to set LPTICKER frequency to RTC_CLOCK/4 */
```

Comment:
Maybe we could add here something like:

#if (MBED_CONF_TARGET_RTC_CLOCK_SOURCE == USE_RTC_CLK_HSE)
#warning "not recommended" ?

or #error ?

Reply:
Good idea. I'm adding this.

Reply:
Done.